### DISCUSSION TITLE
Course 'Blockchain Basics' still talks about PoW

### DISCUSSION THREAD

**Discussion by expiredCookie** [Cyfrin/foundry-full-course-cu#2118] [General]
Hi,
I noticed that the first course 'Blockchain Basics' it's still teaching all the info about Pow and not Pos regarding blocks and hash.

**Comment by EngrPips** [comment]
Hello @expiredCookie, Do you think that is not right?

**Comment by expiredCookie** [comment]
I saw that later on there's a update on PoS also, I was just surprised in the beginning.
But it's always good knowing how PoW works 👍

**Reply by EngrPips** [reply]
Indeed
